llvm/test/CodeGen/RISCV/GlobalISel/alu-roundtrip.ll

```diff
@@ -348,7 +348,9 @@ define i32 @sdiv_i32(i32 %a, i32 %b) {
 ;
 ; RV64IM-LABEL: sdiv_i32:
 ; RV64IM:       # %bb.0: # %entry
-; RV64IM-NEXT:    divw a0, a0, a1
+; RV64IM-NEXT:    sext.w a0, a0
+; RV64IM-NEXT:    sext.w a1, a1
+; RV64IM-NEXT:    div a0, a0, a1
```

Comment:
For now, should we look one level down to see if we know operands are sign extended?

Reply:
What if we look at it during custom legalization?

Reply:
> Edit: I think thats what you mean by custom W nodes?

Yes. In order to do something during custom legalization we would need to convert it to a custom opcode.